src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -212,19 +212,29 @@ function PIASubHeader({
   ]);
 
   const showEditButton = () => {
-    // we may revisit this part later for standard PIA
+    // This section may need further review for standard PIA
+
+    // Check if the privilege for showing the edit button is available
+    const showEditButtonPrivilege =
+      getUserPrivileges(pia)?.Pages?.review.showEditButton || false;
```

Comment:
This should default to true if showEditButton prop is optional

Reply:
Also the "page" shouldn't be hard coded as review. We should come up with a way to see what the privileges are for the current page.

Reply:
We should also probably put showEditButton inside of params and keep everything outside of params specific to the page as a whole like if the use has access to it